### PR TITLE
Release v5.2.0 rc1

### PR DESCRIPTION
**Pull request by nuttycom**

### CODE REVIEW

**Review by daira — APPROVED** [review]
utACK

**Review by therealyingtong — APPROVED** [review]
utACK

File: doc/release-notes/release-notes-5.2.0-rc1.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+This release makes several changes to improve the performance of wallet operations.
+These include:
+
+- We now parallelize and batch trial decryption of Sapling outputs.
+
+- We now prune witness data in the wallet for notes spent more than 100 blocks
+  in the past, so that we can avoid unnecessarily updating those witnesses.
+  In order to take advantage of this performance improvement, users will need
+  to start their nodes with `-rescan` the first time .
```
Comment (ebfull) [inline doc/release-notes/release-notes-5.2.0-rc1.md:29]: A little more clear would be:

> `-rescan` once to reset the wallet

**Review by ebfull — APPROVED** [review]

File: doc/release-notes/release-notes-5.2.0-rc1.md (hunk 1 above)
Comment (nuttycom) [reply doc/release-notes/release-notes-5.2.0-rc1.md:29]: I had been typing a more complete comment got interrupted in the middle and didn't complete it. Hence the space before the period. I'll fix it before the release.

**Pull request merged**